toolkit/do.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
It feels like this logic could be converted to a new target in our tooling instead of being kept as a separate package. What do you think?